### DISCUSSION TITLE
[Help]: General error after login on version 5.12.7

### DISCUSSION THREAD

**Discussion by AndreLFSMartins** [mbentley/docker-omada-controller#370] [Help] [answered]
### Controller Version

5.12.7

### Describe Your Issue or Question

After I login in into the Controller I'm getting general error message and no site appears.

If I restore a backup or a configuration it come back to work, but as soon as I restart the docker it fails again. I have deleted the data directory at the docker let it recreate the data, imported the config and it fails again. 

Any idea how to solve it ?

### Expected Behavior

It would start

### Steps to Reproduce

Just log in 

### How You're Launching the Container

```plain
docker run
```

### Container Logs

```plain
no
```

**Comment by mbentley** [comment, marked as answer]
I can’t help without the requested info besides guessing. Try pulling the image again, maybe related to the issue from https://github.com/mbentley/docker-omada-controller/issues/366
